Lecture-6/Task2/l6task2/AscendingFirstAgeComparator.java

```diff
@@ -0,0 +1,10 @@
+package l6task2;
+
+import java.util.Comparator;
+
+public class AscendingFirstAgeComparator implements Comparator<Student> {
+    @Override
+    public int compare(Student student1, Student student2) {
+        return Integer.valueOf(student1.getAge()).compareTo(Integer.valueOf(student2.getAge()));
```

Comment:
No need to use `valueOf` to convert int to String.  getAge()` returns `int` which should be compared
`return Integer.compare(age1, age2)`

Reply:
done